internal/services/containerapps/container_app_resource_test.go

```diff
@@ -472,6 +472,99 @@ resource "azurerm_container_app" "test" {
 `, r.templatePlusExtras(data), data.RandomInteger, revisionSuffix)
 }
 
+func (r ContainerAppResource) completeWithNoIngressTrafficWeights(data acceptance.TestData, revisionSuffix string) string {
```

Comment:
I missed this: you'll want to add a function using this test config:
```
func TestAccContainerAppResource_completeWithSidecar(t *testing.T) {
	data := acceptance.BuildTestData(t, "azurerm_container_app", "test")
	r := ContainerAppResource{}
	data.ResourceTest(t, r, []acceptance.TestStep{
		{
			Config: r.completeWithSidecar(data, "rev1"),
			Check: acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).ExistsInAzure(r),
			),
		},
		data.ImportStep(),
	})
}
````

Reply:
Thanks @katbyte I added the `TestAccContainerAppResource_completeWithNoIngressTrafficWeights` function to `container_app_resource_test.go` file.